ios/Podfile.lock

```diff
@@ -951,4 +951,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 3fd9f605d2ca061c942789de32928fa0ba430aa8
 
-COCOAPODS: 1.11.3
+COCOAPODS: 1.15.0
```

Comment:
Please revert it. 1.15.0 has changes that won't be working with the latest React Native.
Please install 1.11.3 in your local machine